packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -6,96 +6,104 @@ import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
 /**
  * @title L2OutputOracle
  * @notice The L2 state is committed to in this contract
- * The payable keyword is used on appendL2Output to save gas on the msg.value check.
- * This contract should be deployed behind an upgradable proxy
+ *         The payable keyword is used on appendL2Output to save gas on the msg.value check.
+ *         This contract should be deployed behind an upgradable proxy
  */
 // slither-disable-next-line locked-ether
 contract L2OutputOracle is Ownable {
+    /*********
```

Comment:
I don't believe the contracts have the section headers elsewhere in the codebase